blockchain/metrics.go

```diff
@@ -0,0 +1,81 @@
+package blockchain
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/discard"
+	prometheus "github.com/go-kit/kit/metrics/prometheus"
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+)
+
+// Metrics represents the blockchain metrics
+type Metrics struct {
+	// Gas Price Average
+	GasPriceAvg metrics.Histogram
+	// Gas used
+	GasUsed metrics.Histogram
```

Comment:
Adding a `Gas` for counting total gas here, it would be helpful when DAPP developer try to get more reasonable estimation.

Reply:
`GasCountAverage` is meaning-less.
What I suggested is the total gas each block transactions sum up.